plugins/LilyPad/XInputEnum.cpp

```diff
@@ -268,9 +270,9 @@ void EnumXInputDevices()
 {
     wchar_t temp[30];
     if (!pXInputSetState) {
-        // Also used as flag to indicute XInput not installed, so
+        // XInput not installed, so
         // don't repeatedly try to load it.
```

Comment:
Maybe have the comment on a single line instead of two.

Personally I would split the commit into 2 (bool change, call order change).

Reply:
Do you want me to split it and force push to this PR?

Reply:
I think it'd be better to.